lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -813,3 +812,59 @@ def get_next_line(fh):
         # Discard the rest of the line
         fh.readline()
     return line.strip()
+
+
+class VtkXml(GenericXml):
+    """Format for defining VTK (XML based) and its sub-datatypes. https://docs.vtk.org/en/latest/design_documents/VTKFileFormats.html"""
+
+    edam_format = "edam:format_2332"
+    file_ext = "vtkxml"
+
+    # The same MetadataElements are also available for legacy VTK datatypes.
+    MetadataElement(name="vtk_version", default=None, desc="Vtk version", readonly=True, optional=True, visible=True)
```

Comment:
We don't standardise those names, maybe we should? Maybe just `version`?

Reply:
The name has been chosen intentionally to match the metadata names of the legacy datatypes https://github.com/galaxyproject/galaxy/blob/ff0530c8a1a7acb2b0c1f3b7d1d8c5ac2bd3ad35/lib/galaxy/datatypes/constructive_solid_geometry.py#L198

Reply:
Ok, go for it!